views/pages/ember.hbs

```diff
@@ -0,0 +1,223 @@
+    </p>
+
+    <p>
+        The <code>\{{format-message}}</code> helper formats a translated message written in the <a href="http://userguide.icu-project.org/formatparse/messages">ICU Message</a> syntax, which is used by professional translators. The message format supports placeholders, plus choosing different strings based on pluralization, gender, or other criteria.
+    </p>
+
+    {{> integrations/see-guide}}
+
+    {{~> example examples.message}}
+
+    <p>
+        In the example above, we use <code>(intl-get "messages.photos")</code> as a subexpression to delegate the lookup process for <code>"message.photos"</code>. It is equivalent to the accessing the value from <code>this.intl.messages.photos</code>, but it will provide descriptive error messages when accessing a property that is <code>undefined</code>, unlike Handlebars built-in data channel access syntax <code>@</code>. Check the <strong>RENDER</strong> tab to see the details.
+    </p>
+
+    <p class="note">
+        <b>Note:</b> You can provide the <code>messages</code> object when rendering the top-level template, and <code>messages</code> &mdash; just like <code>locales</code> &mdash; will be propagated to any partial or helper.
+    </p>
+
+    <p>
+        Additionally, you can use <a href="#custom-formats">named formats</a> as described below, to specify a set of named format options to use in your messages, e.g: <code>{someNum, number, USD}</code> and <code>{someDate, date, short}</code> or <code>{someTime, time, long}</code>.
+    </p>
+
+    <h3>Format Messages That Contain HTML</h3>
+
+    <p>
+        If a message string contains HTML, you can use the <code>\{{format-html-message}}</code> helper, which has the exact same API as <code>\{{format-message}}</code> but it will not escape the HTML in the message's text. However, it will escape all the values the message if formatted with, but  remember to be cautious of untrusted user input to prevent XSS injections.
+    </p>
+
+    <h2 id="custom-formats">Define Your Own Custom Formats</h2>
+
+    <p>
+        Handlebars Intl allows you to define named formats that you can be used throughout your entire application or within a template and its partials. These named formats can be specified for <code>date</code>, <code>time</code>, <code>number</code> and <code>relative</code> format types. The following example illustrates how these custom named formats work.
+    </p>
+</section>
+
+{{#section-for 'footer'}}
+    {{! renders into the footer section of view/layouts/main.hbs}}
+    <script src="/vendor/jquery/jquery{{min}}.js"></script>
+    <script src="/vendor/ember/ember{{min}}.js"></script>
+    <script src="/vendor/ember-intl/ember-intl.js"></script>
```

Comment:
We could also move these into the "vendor" partial wrapped in a condition that checks the name of the current page or something so it only applies to Ember. Whatever you think it easiest to reason about.